usaspending_api/common/validator/helpers.py

```diff
@@ -87,7 +87,7 @@ def _verify_float_value(value):
 
 
 def validate_array(rule):
-    rule["min"] = rule.get("min") or 1
+    rule["min"] = 1 if rule.get("min") is None else rule.get("min")  # Allows empty arrays
```

Comment:
Do we want empty arrays? I think the original intention was to not allow a key with an empty array. simply block the request and remove the key & empty array

Reply:
We have a few places in the frontend that are supplying empty arrays as filters to the downloads. Can handle those in the request if we don't want to make this change.

Reply:
Discussed in slack. Plan to resolve in the future